root/frontend/src/components/Day/Day.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { format } from 'date-fns';
+
+import useStyles from './Day.style';
+import Button from '../Button/Button';
+
+type PropTypes = {
+  isSelected: boolean,
+  isToday?: boolean,
+  date: Date,
+};
+
+const generateDateURL = (date: Date): string => `../Workouts?date=${format(date, 'MMddyyyy')}`;
+
+function Day({ isSelected, isToday, date }: PropTypes) {
+  const styles = useStyles();
+  const navigate = useNavigate();
+
+  let dayStyle;
+
+  if (isToday) dayStyle = styles.today;
+  else dayStyle = isSelected ? styles.selectedDay : styles.unSelectedDay;
```

Comment:
Just to confirm for consistency, `if {} else {}` is used for multi-line operations and we adopt this approach ^ for single line operations?

Reply:
I don't really have a rule that I was following for using an `if/else` vs a single line conditional. Generally, I like to use single line conditionals for operations that return a simple item (in this case either `styles.selectedDay` or `styles.unSelectedDay`) as it helps alleviate some clutter that `if` statements generate while still remaining readable. I do think that this line in particular can be seen as being simpler to read and more difficult to read, so let me know what you think. It may be better to only use `if` statements here.

Reply:
No, I think the way you have it set up right now is fine. I guess moving forward we can follow this structure for single and multi-line operations.